client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2156,6 +2156,20 @@ Undocumented.prototype.transferStatus = function( siteId, transferId ) {
 };
 
 /**
+ * Get user suggestions for a site.
+ *
+ * @param {int}       siteId            The site ID
+ * @returns {Promise}
+ */
+Undocumented.prototype.getUserSuggestions = function( siteId ) {
+	debug( '/users/suggest/ query' );
+
+	return this.wpcom.users().suggest( {
```

Comment:
In the action we can probably just use wpcom directly. Undocumented is generally reserved for endpoints that haven't made it into [wpcom.js](https://github.com/Automattic/wpcom.js)
```
import wpcom from 'lib/wp';

return this.wpcom.users().suggest( { //...
```